node/Cargo.toml

```diff
@@ -12,50 +12,90 @@ edition = "2018"
+sp-offchain = { git = 'https://github.com/paritytech/substrate', branch = "cmichi-tmp-experiment" }
+sp-runtime = { git = 'https://github.com/paritytech/substrate', branch = "cmichi-tmp-experiment" }
+sp-session = { git = 'https://github.com/paritytech/substrate', branch = "cmichi-tmp-experiment" }
+sp-timestamp = { git = 'https://github.com/paritytech/substrate', branch = "cmichi-tmp-experiment" }
+sp-transaction-pool = { git = 'https://github.com/paritytech/substrate', branch = "cmichi-tmp-experiment" }
+
+# Cumulus dependencies
+cumulus-client-consensus-aura = { git = 'https://github.com/paritytech/cumulus', branch = "hc-contract-experiment-patched-v0.9.9" }
+cumulus-client-consensus-common = { git = 'https://github.com/paritytech/cumulus', branch = "hc-contract-experiment-patched-v0.9.9" }
+cumulus-client-collator = { git = 'https://github.com/paritytech/cumulus', branch = "hc-contract-experiment-patched-v0.9.9" }
+cumulus-client-cli = { git = 'https://github.com/paritytech/cumulus', branch = "hc-contract-experiment-patched-v0.9.9" }
+cumulus-client-network = { git = 'https://github.com/paritytech/cumulus', branch = "hc-contract-experiment-patched-v0.9.9" }
+cumulus-client-service = { git = 'https://github.com/paritytech/cumulus', branch = "hc-contract-experiment-patched-v0.9.9" }
+cumulus-primitives-core = { git = 'https://github.com/paritytech/cumulus', branch = "hc-contract-experiment-patched-v0.9.9" }
+cumulus-primitives-parachain-inherent = { git = 'https://github.com/paritytech/cumulus', branch = "hc-contract-experiment-patched-v0.9.9" }
+
+# Polkadot dependencies
+polkadot-cli = { git = 'https://github.com/paritytech/polkadot', branch = "hc-contract-experiment-patched-v0.9.9" }
+polkadot-parachain = { git = 'https://github.com/paritytech/polkadot', branch = "hc-contract-experiment-patched-v0.9.9" }
+polkadot-primitives = { git = 'https://github.com/paritytech/polkadot', branch = "hc-contract-experiment-patched-v0.9.9" }
+polkadot-service = { git = 'https://github.com/paritytech/polkadot', branch = "hc-contract-experiment-patched-v0.9.9" }
+polkadot-test-service = { git = 'https://github.com/paritytech/polkadot', branch = "hc-contract-experiment-patched-v0.9.9" }
+
+# Contracts specific packages
+pallet-contracts = { git = "https://github.com/paritytech/substrate", package = "pallet-contracts", default-features = false , branch = "cmichi-tmp-experiment" }
+pallet-contracts-primitives = { git = "https://github.com/paritytech/substrate", package = "pallet-contracts-primitives", default-features = false , branch = "cmichi-tmp-experiment" }
+pallet-contracts-rpc = { git = "https://github.com/paritytech/substrate", package = "pallet-contracts-rpc", default-features = false , branch = "cmichi-tmp-experiment" }
+pallet-contracts-rpc-runtime-api = { git = "https://github.com/paritytech/substrate", package = "pallet-contracts-rpc-runtime-api", default-features = false , branch = "cmichi-tmp-experiment" }
```

Comment:
Do we need all of those in the client? Don't we just need `pallet-contracts-rpc` here?